### ISSUE TITLE
Wish: a DHGR slideshow viewer integrated with the surl server

### ISSUE THREAD

**Issue by xandark** [colinleroy/a2tools#2]
Very cool tools! I see you have imageviewer.dsk that can show HGR images, but could it be modified to show DHGR ones? That would be great, especially with all of the fantastic work that [ii-pix](https://github.com/KrisKennaway/ii-pix) can do.

However, I have to confess that my original thought was to have a slideshow of a directory of .jpg, .gif, etc images from the ftp surl server or wherever surl can fetch files. The images would be pre-processed on the server with ii-pix to convert to DHGR and then sent to the Apple II and displayed. I know, this is not as technically ambitious as streaming video(!), but it would be very cool to use my //c as a slideshow viewer at its best color abilities.

**Comment by colinleroy** [comment]
Yes, it's been a while I want to add a DHGR command to the surl protocol, it's been put on the backburner for now, but it may come at some point!

**Comment by xandark** [comment]
Awesome! Okay, I'll look out for it.

**Comment by colinleroy** [comment]
Hi @xandark !
So, that's not DHGR, only simple HGR.
But, in the latest release (https://github.com/colinleroy/a2tools/releases/tag/v21.0.0), Wozamp can be used as a slideshow application. Basically, point it to an FTP directory containing pictures and/or subdirectories with pictures, and use "Play All" or "Random".

**Comment by colinleroy** [comment]
hello @xandark ,

I believe the pre-release at https://github.com/colinleroy/a2tools/releases/tag/v24.0.1 fulfills this wish. You can use Wozamp to slideshow whole pictures directories hosted on ftp servers, or Ammonoid to view DHGR pictures on ProDOS volumes, including VSDrives. (no automatic slideshow there though)

Hope you'll like it :)

**Comment by xandark** [comment]
woohoo! awesome!
